docs/tutorial-forms.rst

```diff
@@ -111,6 +111,23 @@ Adding Additional Attributes
 ----------------------------
```

Comment:
@jrief I saw in the logs you added this line but there is no docs under it, shouldnt it be removed? Or completed?